shared/devices/paper-key/container.tsx

```diff
@@ -22,8 +22,6 @@ const mergeProps = (stateProps, dispatchProps) => ({
 })
 
 // @ts-ignore codemode issue
-export default Container.connect<OwnProps, _, _, _, _>(
-  mapStateToProps,
-  mapDispatchToProps,
-  mergeProps
-)(PaperKey)
+export default Container.connect<OwnProps, _, _, _, _>(mapStateToProps, mapDispatchToProps, mergeProps)(
```

Comment:
We can remove the generics and the ts-ignore

Reply:
lets do taht as another pass